crypto/x509/v3_addr.c

```diff
@@ -985,7 +990,10 @@ static void *v2i_IPAddrBlocks(const struct v3_ext_method *method,
         switch (delim) {
         case '/':
             prefixlen = (int)strtoul(s + i2, &t, 10);
-            if (t == s + i2 || *t != '\0') {
+            if (t == s + i2
+                    || *t != '\0'
+                    || prefixlen > (length * 8)
+                    || prefixlen < 1) {
```

Comment:
Again, I think a prefix length of 0 should be permitted. I'm not sure it's worth it to add further checks to ensure the address is all zeroes.

Reply:
Fixed